plugin-parser/mlops-plugin-parser.cc

```diff
@@ -128,16 +246,59 @@ _parse_json (JsonNode *node, mlsvc_json_type_e json_type, const gchar *app_info)
             continue;
```

Comment:
Need to consider return FALSE and handle the error case in `continue` lines

Reply:
thanks, i update it.